src/pages/donate.astro

```diff
@@ -103,6 +109,7 @@ import { resolvePath } from '@/utils/helpers.js'
 .c-container-center {
   text-align: center;
   margin: 6rem auto 0 auto;
+  padding: 0 1rem;
```

Comment:
Boy-scouting this spacing bug:
<img src='https://github.com/user-attachments/assets/17a848bd-a667-48bc-936d-caafeddee611' width='390'>